src/test/data/core_api_snapshot.json

```diff
@@ -0,0 +1,272 @@
+{
+  "ActionURL": [
```

Comment:
So is the process that you generate or update this file based on what is in the LABKEY object and then run the test to compare what is missing / added/ removed?

Reply:
Nevermind, I just read the comment in the next file for the generate js file. Cool